bitnami/wavefront/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+name: wavefront
+description: Chart for Wavefront Collector for Kubernetes
+appVersion: 1.2.1
+version: 0.1.0
+keywords:
+  - metric
+  - monitoring
+  - observability
+  - alerting
+home: https://www.wavefront.com
+sources:
+  - https://github.com/bitnami/bitnami-docker-wavefront-collector
+  - https://github.com/bitnami/bitnami-docker-wavefront-proxy
```

Comment:
I'd add these links too:
- https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes
- https://github.com/wavefrontHQ/wavefront-proxy